.github/workflows/clang-format.yml

```diff
@@ -0,0 +1,20 @@
+name: clang-format-lint
+
+on:
+  push:
+    branches: [ master, develop ]
```

Comment:
Is there any reason to limit the branches?

Reply:
There's some limit to hours spent on github actions for free accounts. Plus I am not sure if you can wildcard the branch name. Master/develop are the only branches which accept downstream changes anyway

Reply:
Okay, got it.